src/ldc/intrinsics.di

```diff
@@ -19,69 +19,19 @@ else
     static assert(false, "This module is only valid for LDC");
 }
 
-version (LDC_LLVM_309)
-{
-}
-else version (LDC_LLVM_400)
-{
-    version = INTRINSICS_FROM_400;
-}
-else version (LDC_LLVM_500)
-{
-    version = INTRINSICS_FROM_400;
-    version = INTRINSICS_FROM_500;
-}
-else version (LDC_LLVM_600)
-{
-    version = INTRINSICS_FROM_400;
-    version = INTRINSICS_FROM_500;
-    version = INTRINSICS_FROM_600;
-}
-else version (LDC_LLVM_700)
-{
-    version = INTRINSICS_FROM_400;
-    version = INTRINSICS_FROM_500;
-    version = INTRINSICS_FROM_600;
-    version = INTRINSICS_FROM_700;
-}
-else version (LDC_LLVM_701)
-{
-    version = INTRINSICS_FROM_400;
-    version = INTRINSICS_FROM_500;
-    version = INTRINSICS_FROM_600;
-    version = INTRINSICS_FROM_700;
-}
-else version (LDC_LLVM_800)
-{
-    version = INTRINSICS_FROM_400;
-    version = INTRINSICS_FROM_500;
-    version = INTRINSICS_FROM_600;
-    version = INTRINSICS_FROM_700;
-    version = INTRINSICS_FROM_800;
-}
-else version (LDC_LLVM_900)
-{
-    version = INTRINSICS_FROM_400;
-    version = INTRINSICS_FROM_500;
-    version = INTRINSICS_FROM_600;
-    version = INTRINSICS_FROM_700;
-    version = INTRINSICS_FROM_800;
-    version = INTRINSICS_FROM_900;
-}
-else version (LDC_LLVM_1000)
-{
-    version = INTRINSICS_FROM_400;
-    version = INTRINSICS_FROM_500;
-    version = INTRINSICS_FROM_600;
-    version = INTRINSICS_FROM_700;
-    version = INTRINSICS_FROM_800;
-    version = INTRINSICS_FROM_900;
-    version = INTRINSICS_FROM_1000;
-}
-else
-{
-    static assert(false, "LDC LLVM version not supported");
-}
+     version (LDC_LLVM_309)  enum LLVM_version =  309;
+else version (LDC_LLVM_400)  enum LLVM_version =  400;
+else version (LDC_LLVM_500)  enum LLVM_version =  500;
+else version (LDC_LLVM_600)  enum LLVM_version =  600;
+else version (LDC_LLVM_700)  enum LLVM_version =  700;
+else version (LDC_LLVM_701)  enum LLVM_version =  701;
+else version (LDC_LLVM_800)  enum LLVM_version =  800;
+else version (LDC_LLVM_900)  enum LLVM_version =  900;
+else version (LDC_LLVM_1000) enum LLVM_version = 1000;
+else version (LDC_LLVM_1100) enum LLVM_version = 1100;
+else static assert(false, "LDC LLVM version not supported");
+
+enum LLVM_atleast(int major) = (LLVM_version >= major * 100);
```

Comment:
Intentionally public, so that the user is free to use this `LDC_LLVM_xxx` => `int` mapping too.